### DISCUSSION TITLE
Copilot in Visual Studio 2022，did not show suggestion in the Debug mode

### DISCUSSION THREAD

**Discussion by MycroftDevton** [community/community#144275] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I would like to know if Copilot suggestions are disabled by default in debug mode and if they return to normal after closing debug mode?
![image](https://github.com/user-attachments/assets/1d545ca7-1ce2-4f03-9d3f-12101b63d56e)、
![image](https://github.com/user-attachments/assets/75cf635b-67ca-49bf-94ca-ea75b54b1029)

**Comment by pratham-Lashkari** [comment, marked as answer]
Completions are off by default in debug mode. You can turn them back on in Tools > Options > IntelliCode > Advanced.
![371308670-e48f8402-3bd2-4ed1-9c06-9330a1fa9eb9](https://github.com/user-attachments/assets/d098c398-cef3-4145-8474-5c74f4dd6ff6)

**Reply by MycroftDevton** [reply]
Help me a lot, thank you.
